frontend/src/validators.js

```diff
@@ -145,4 +145,18 @@ export default {
     if (phoneNumber.length == 10 && /^[0-9]+$/g.test(phoneNumber)) return true
     return "Dix chiffres numériques attendus"
   },
+  decimalPlaces(max) {
+    return (input) => {
+      const number = Number(input)
```

Comment:
We currently don't seem to manage the comma as a decimal separator. This is probably out of scope of this PR but let's have a few words on it tomorrow.

![image](https://user-images.githubusercontent.com/1225929/189607490-26d5544c-8f95-48a0-a909-e00095118a3d.png)